### ISSUE TITLE
Could not connect to the remote Jenkins when password contains #

### ISSUE THREAD

**Issue by VitekVlcek-Broadcom** [tabeyti/jenkins-jack#41]
When a password starts with `#` then I get `Could not connect to the remote Jenkins`

Developer console reveals
```
Using the following URI for Jenkins client: http://abc123:#password+-@jenkins3.acme.net
console.ts:137 [Extension Host] Refreshing Pipeline step auto-completions.
console.ts:137 [Extension Host] RequestError: Error: getaddrinfo ENOTFOUND abc123 
   at new RequestError (/home/XXX/.vscode-server/extensions/tabeyti.jenkins-jack-1.1.0/node_modules/request-promise-core/lib/errors.js:14:15)	
   at Request.plumbing.callback (/home/XXX/.vscode-server/extensions/tabeyti.jenkins-jack-1.1.0/node_modules/request-promise-core/lib/plumbing.js:87:29)
```

There is a workaround:
When I replace `#` with `%23` in  `password` in `settings.json`, then I can connect.

**Comment by tabeyti** [comment]
Hey @VitekVlcek-Broadcom 

This should now be [fixed in 1.1.1](https://github.com/tabeyti/jenkins-jack/blob/master/CHANGELOG.md#fixed). If you could please validate the fix for me, I'd appreciate it.

**Comment by VitekVlcek-Broadcom** [comment]
yes, 1.1.1 fixes it.
